readme.md

```diff
@@ -23,17 +23,29 @@ const Emittery = require('emittery');
 
 const emitter = new Emittery();
 
+const myEvent = Symbol('my symbol event');
```

Comment:
Could use a better example. This doesn't really show how Symbols can be useful as event names.